integration/framework/framework.go

```diff
@@ -209,37 +244,89 @@ func (self *realFramework) ClientV2() *v2.Client {
 	return self.cadvisorClientV2
 }
 
-func (self dockerActions) RunPause() string {
-	return self.Run(DockerRunArgs{
+func (self rktActions) RuntPause() string {
```

Comment:
I think it would be good to split this file up, and put the rkt-specific code in rkt.go